### ISSUE TITLE
GUI elements are not scaled properly

### ISSUE THREAD

**Issue by Trass3r** [tonihele/OpenKeeper#343]
When using higher-resolution images:
![screenshot](https://user-images.githubusercontent.com/404623/66657981-d826e300-ec40-11e9-9e88-7df79c726248.jpg)

**Comment by tonihele** [comment]
Did you generate these by some machine learning solution? If so, can it handle the transparencies? Also we could generate the test data with the same lib then :)

**Comment by Trass3r** [comment]
I tested Gigapixel. With transparency you get color fringes.

**Comment by tonihele** [comment]
I used https://github.com/xinntao/ESRGAN awhile back. Otherwise I got very nice results, but it didn't support transparencies. Got amazing detail to Reaper (https://cdn.knightlab.com/libs/juxtapose/latest/embed/index.html?uid=db515096-1e79-11e9-9dba-0edaf8f81e27).

Normal maps... Yeah you need to be up close and personal with them. Also we are only currently using only ambient light. I guess that contributes to the fact that everything looks a bit flat. We have an experimental branch (https://github.com/tonihele/OpenKeeper/tree/experimental-light) with real lights that was used by a artist for creating custom graphics with normal maps. The branch hasn't been updated in awhile, but it still of course works to see the graphics.

**Comment by Trass3r** [comment]
> I used https://github.com/xinntao/ESRGAN awhile back. Otherwise I got very nice results, but it didn't support transparencies.

Most don't. You end up having to upscale the alpha separately and then recombine.

Got amazing detail to Reaper (https://cdn.knightlab.com/libs/juxtapose/latest/embed/index.html?uid=db515096-1e79-11e9-9dba-0edaf8f81e27).

Btw there is no launcher script for the model viewer.

> we are only currently using only ambient light. I guess that contributes to the fact that everything looks a bit flat.

Yes I noticed that as well. Of course normal maps don't make a difference without at least diffuse lighting.

We have an experimental branch (https://github.com/tonihele/OpenKeeper/tree/experimental-light) with real lights that was used by a artist for creating custom graphics with normal maps. The branch hasn't been updated in awhile, but it still of course works to see the graphics.

I rebased it but too much has changed in the code for it to just work: https://github.com/Trass3r/OpenKeeper/tree/experimental-light

**Comment by tonihele** [comment]
> I rebased it but too much has changed in the code for it to just work: https://github.com/Trass3r/OpenKeeper/tree/experimental-light

Yeah, I think the code was before our huge restructure. I can try to bring it to life in few days. There wasn't huge changes that I made back then.

**Comment by Trass3r** [comment]
Yeah it should definitely improve the visuals.
